docs/make.jl

```diff
@@ -2,21 +2,21 @@ using SymbolServer
 using Documenter
 
 makedocs(;
-    modules=[SymbolServer],
-    authors="Julia VSCode",
-    repo="https://github.com/julia-vscode/SymbolServer.jl/blob/{commit}{path}#L{line}",
-    sitename="SymbolServer.jl",
-    format=Documenter.HTML(;
-        prettyurls=prettyurls = get(ENV, "CI", nothing) == "true",
+    modules = [SymbolServer],
+    authors = "Julia VSCode",
+    repo = "https://github.com/julia-vscode/SymbolServer.jl/blob/{commit}{path}#L{line}",
+    sitename = "SymbolServer.jl",
+    format = Documenter.HTML(;
+        prettyurls = prettyurls = get(ENV, "CI", nothing) == "true"
         # canonical="https://www.julia-vscode.org/SymbolServer.jl",
         # assets=String[],
     ),
-    pages=[
+    pages = [
         "Home" => "index.md",
         "Syntax Reference" => "syntax.md",
-    ],
+    ]
 )
 
 deploydocs(;
-    repo="github.com/julia-vscode/SymbolServer.jl",
+    repo = "github.com/julia-vscode/SymbolServer.jl"
```

Comment:
This comma should not be removed (it should neither add nor remove commas in this situation).